### DISCUSSION TITLE
Migration Changes not usable

### DISCUSSION THREAD

**Discussion by danielsumah** [prisma/prisma#15497] [Q&A] [answered]
I had a model/schema in this format

![image](https://user-images.githubusercontent.com/52764894/192048851-4e6a4741-be1b-415d-8ad7-117dba17989e.png)

i update the amount to be a float and added transactionDate column
![image](https://user-images.githubusercontent.com/52764894/192048964-36cb8062-e5c3-43fd-b24b-fc1acc7fe3a0.png)

After making migrations, the amount still appears as a string and the newly added transactionDate is not there
![image](https://user-images.githubusercontent.com/52764894/192049218-0eb9feb9-cea8-40ae-a84b-54f3059c7325.png)

![image](https://user-images.githubusercontent.com/52764894/192049266-48882c52-6368-4716-9239-7488d85ff358.png)

A screenshot of my dto
![image](https://user-images.githubusercontent.com/52764894/192049355-8e0f0618-cd33-440a-b9df-99a021ee172d.png)

Please, what can be causing this error ?

**Comment by austincrim** [comment, marked as answer]
Hey @danielsumah 👋,

Can you try re-running `npx prisma generate` and restarting Visual Studio Code?

**Reply by dsumah** [reply]
Thank you so much @austincrim ..

**Reply by danielsumah** [reply]
Issue solved with your recommendation(s) @austincrim .. I learned that `npx prisma generate` helps to create typescript types for my schema

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
